src/constants/order_status.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class OrderStatus(Enum):
+    NEW_ORDER = "new_order"
```

Comment:
Could briefly explain the lifecycle of an order?

Reply:
The lifecycle will be:

when an order is create its first status is **"new_order"**, after the order passed through all the checks, like ingredient availability validation,if the order don't passed any of the checks the order will pass to **"cancelled"** status, the order is passed to **"placed_order"** status, after that an available chef take the order and change its status to **"in_process"**. when the order runs out of preparation time the chef put the order in **"complete**" status.

Reply:
Nice thanks, but could you write this in the code?